src/commandHandlers/standup.ts

```diff
@@ -1,18 +1,25 @@
-import { MessageEmbed } from 'discord.js';
+import { MessageEmbed, Message } from 'discord.js';
 
 /**
  * This command lets the user create a message for the daily standup in the community. For more information on
  * standups, check the link.
  *
  * {link https://www.atlassian.com/agile/scrum/standups}
  */
-export const command = async (arg: string, embed: MessageEmbed) => {
+export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
```

Comment:
Should `message` be added to all the other commands? Just thinking out loud

Reply:
Yeah I was thinking about that too 🙂. I figured if I added the parameter to the other handlers, the linter wouldn't like that it's not used:
![imagem](https://user-images.githubusercontent.com/18630253/83975414-c7bc2a80-a8eb-11ea-9454-34ef0595b446.png)

... on the other side, by being there, a new developer doesn't need to read the docs (or search the codebase) **to know** the `message` parameter is indeed **passed**. So perhaps we could add it.

Reply:
Yeah I not sure what is best. I think that isn't the linter though, that is vscode. But yes when the linter is working properly, it will probably complain 🤦